src/Umbraco.Web.UI.Client/gulp/tasks/dependencies.js

```diff
@@ -232,11 +232,6 @@ function dependencies() {
             "name": "typeahead.js",
             "src":  ["./node_modules/typeahead.js/dist/typeahead.bundle.min.js"],
             "base": "./node_modules/typeahead.js/dist"
-        },
-        {
-            "name": "underscore",
```

Comment:
Does this mean underscore js will not be shipped with the backoffice anymore ? This will be massive breaking change for package developers. So this needs to be made clear way in advance. And have some kind of upgrade instructions for package developers available

Reply:
I believe the point of this PR is to START to decouple Umbraco from its dependency, as many functions are now natively supported. Once Umbraco Core no longer needs it, a deprication strategy can be formed. On Twitter yesterday it was suggest to create a deprication layer that proxies the new methods outputting warnings to the console log to upgrade. Ultimately the intent is not to just “rip it out” but to just make a start to gradually remove these unnecessary dependencies.

Reply:
Yeh I was maybe a bit too aggressive on the removal - so this will get rolled back @dawoe & @mattbrailsford after finding out package devs use it more than I thought.

So goal of this will be to remove our own usage of underscore.js